io/disk/arcconf/arcconf_cntl_oper.py

```diff
@@ -83,7 +83,7 @@ def setUp(self):
                 http_repo = "%s%s.rpm" % (self.http_path, self.tool_name)
                 self.repo = self.fetch_asset(http_repo, expire='10d')
                 cmd = "rpm -ivh %s" % self.repo
-            if process.system(cmd, ignore_status=True, shell=True) == 0:
+            if process.system(cmd, ignore_status=True, shell=True) != 0:
```

Comment:
Well I wouldn't call this a big change, would you? (the purpose of authorship is to mark significant changes like new API, complete reworks etc. This change is only a minor modification which does not require the change of copyright) .

See eg. the first response to http://programmers.stackexchange.com/questions/203014/author-has-inserted-copyright-into-code-with-gnu-public-license-notice-implica